src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/child-place-of-birth.js

```diff
@@ -0,0 +1,116 @@
+      minItems: 1,
+      items: {
+        type: 'object',
+        properties: {
+          state: genericSchemas.genericTextInput,
+          city: genericSchemas.genericTextInput,
+          childStatus: {
+            type: 'object',
+            properties: {
+              biological: genericSchemas.genericTrueFalse,
+              adopted: genericSchemas.genericTrueFalse,
+              notCapable: genericSchemas.genericTrueFalse,
+              stepchild: genericSchemas.genericTrueFalse,
+              dateBecameDependent: genericSchemas.date,
+            },
+          },
+          'view:marriageTypeInformation': {
+            type: 'object',
+            properties: {},
+          },
+          childPreviouslyMarried: {
+            type: 'string',
+            enum: ['Yes', 'No'],
+            default: 'No',
+          },
+          childPreviousMarriageDetails: {
+            type: 'object',
+            properties: {
+              dateMarriageEnded: genericSchemas.date,
+              reasonMarriageEnded: {
+                type: 'string',
+                enum: ['Divorce', 'Death', 'Annulment', 'Other'],
+                default: 'Divorce',
+              },
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+export const uiSchema = {
+  childrenToAdd: {
+    'ui:options': {
+      itemName: 'Child',
+      viewField: childInfo,
+    },
+    'ui:title': "Child's place of birth",
+    items: {
+      state: {
+        'ui:title': 'State (or country if outside the USA)',
+        'ui:required': formData => isChapterFieldRequired(formData, 'addChild'),
+      },
+      city: {
+        'ui:title': 'City or county',
+      },
+      childStatus: {
+        'ui:title': "Your child's status (check all that apply)",
+        biological: {
+          'ui:title': 'Biological',
+        },
+        adopted: {
+          'ui:title': 'Adopted',
+        },
+        notCapable: {
+          'ui:title': 'Not capable of self-support',
+        },
+        stepchild: {
+          'ui:title': 'Stepchild',
+        },
+        dateBecameDependent: _.merge(
+          currentOrPastDateUI('Date stepchild became dependent'),
+          {
+            'ui:options': {
+              expandUnder: 'stepchild',
+              expandUnderCondition: true,
+              keepInPageOnReview: true,
+            },
+          },
+        ),
+      },
+      'view:marriageTypeInformation': {
+        'ui:description': childStatusDescription,
+      },
+      childPreviouslyMarried: {
+        'ui:widget': 'radio',
+        'ui:title': 'Was this child previously married?',
+      },
+      childPreviousMarriageDetails: {
+        'ui:options': {
+          expandUnder: 'childPreviouslyMarried',
+          expandUnderCondition: 'Yes',
+          keepInPageOnReview: true,
+        },
+        dateMarriageEnded: currentOrPastDateUI('When did the marriage end'),
+        reasonMarriageEnded: {
+          'ui:widget': 'radio',
```

Comment:
I think these should be required if a user selects 'yes' to a child being married. Referencing the [full walkthrough](https://xd.adobe.com/view/ffffe536-2ba5-47c6-4c4b-ac1679179973-89ce/screen/e6e102d4-1db1-4851-b63d-6c0d919fea68/Apply-Dependents-Add-Child-2) but should double check with Jim. Looks like `dateMarriageEnded` should at least be required based on the linked mockup. Assuming that if a user selects `other` here, we'll need an additional expandable text field as well.

Reply:
Good catch! Updated this.